src/set.js

```diff
@@ -0,0 +1,59 @@
+'use strict'
+
+export default class Set {
+  constructor(){
+    this.storage = []
+    this.index = 0
+  }
+
+  add(element){
+    if(!this.contains(element)){
+      this.storage[this.index++] = element
+    }
+  }
+
+  contains(element){
+    for(let i = 0; i < this.index; i++){
+      if(this.storage[i] === element){
```

Comment:
Thanks for the feedback, how would a ternary work within the loop? I only want it to return false if its gone through the whole loop comparing the contents of this.storage to the element.